registrations/tasks.py

```diff
@@ -380,7 +373,13 @@ def run(self, registration_id, **kwargs):
                 self.set_risk_status(registration)
 
             self.l.info("Scheduling registration push to Jembi")
-            self.send_to_jembi(registration)
+            jembi_task = BasePushRegistrationToJembi.\
+                get_jembi_task_for_registration(registration)
+            task = chain(
+                jembi_task.si(str(registration.pk)),
+                remove_personally_identifiable_fields.si(str(registration.pk))
```

Comment:
Does this only trigger if the previous task was successful? We might not want to remove the fields if we failed to send the info to Jembi.